LocalDynamoDb/LocalDynamo.cs

```diff
@@ -24,15 +25,15 @@ public LocalDynamo(int portNumber = 8000)
         private static Process Create(int portNumber)
         {
             var processJar = new Process();
-            var arguments = $"-Djava.library.path=./DynamoDBLocal_lib -jar DynamoDBLocal.jar -sharedDb -inMemory -port {portNumber}";
+            var arguments = $"-Djava.library.path=.{Path.DirectorySeparatorChar}DynamoDBLocal_lib -jar DynamoDBLocal.jar -sharedDb -inMemory -port {portNumber}";
 
-            processJar.StartInfo.FileName = "\"" + @"java" + "\"";
+            processJar.StartInfo.FileName = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? "\"" + @"java" + "\"" : "java";;
             processJar.StartInfo.Arguments = arguments;
 
             var rootFolder = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
-            var relativePath = "\\dynamodblocal";
-            string absolutePath = Path.GetFullPath(rootFolder + relativePath);
-            var jarFilePath = absolutePath + "\\DynamoDBLocal.jar";
+            var relativePath = Path.DirectorySeparatorChar + "dynamodblocal";
+            var absolutePath = Path.GetFullPath(rootFolder + relativePath).Replace('\\', Path.DirectorySeparatorChar);
```

Comment:
`Path.Join()` should do all this so we don't need to use `Path.DirectorySeparatorChar` or any string concatenation.